modules/rgbd/src/hash_tsdf.cpp

```diff
@@ -0,0 +1,627 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html
+
+// This code is also subject to the license terms in the LICENSE_KinectFusion.md file found in this
+// module's directory
+#include "hash_tsdf.hpp"
```

Comment:
`precomp.hpp` should be the first include of src/*.cpp files

Reply:
Done. but this file currently does not have any code optimization.

Reply:
.cpp files only, not .hpp files.

Reply:
Okay, done. Is this a preference, or is it that, every header which has optimizations declared requires this?

Reply:
It is some kind of this: https://en.wikipedia.org/wiki/Precompiled_header